sources/api/netdog/test_data/net_config/basic/invalid_interface_config.toml

```diff
@@ -1,4 +1,4 @@
-version = 1
+version = {version}
```

Comment:
I would tend to reach for handlebars here so we don't end up with multiple, similar-looking templating systems.